py/desispec/desi_proc_dashboard.py

```diff
@@ -240,19 +267,15 @@ def main(args):
     #strTable += js_import_str(os.getenv('DESI_DASHBOARD'))
     strTable += js_str()
     strTable += _closing_str()
-    with open(os.path.join(os.getenv('DESI_DASHBOARD'),args.output_name),'w') as hs:
+    with open(os.path.join(os.getenv('DESI_DASHBOARD'),args.output_name+'.html'),'w') as hs:
```

Comment:
Let's change this back to _not_ auto-adding the extension, to keep it the same as it was before.